docs/sphinx/source/conf.py

```diff
@@ -164,12 +162,23 @@
     ],
     "use_edit_page_button": True,
     "show_toc_level": 1,
-    "footer_items": ["copyright", "sphinx-version", "sidebar-ethical-ads"],
-    "left_sidebar_end": [],
-}
+    # "footer_start": [],  # "copyright", "sphinx-version"
+    # "footer_center": [],
+    "footer_end": [],
+    "primary_sidebar_end": [],
```

Comment:
```suggestion
    # "primary_sidebar_end": [],
```
This defaults to ``"primary_sidebar_end": ["sidebar-ethical-ads"]``, and should be commented out for ads to show up. Probably an oversight on my behalf.

Reply:
Don't worry, to be fair I also didn't bother enough to double check your suggestion.